pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -47,6 +49,7 @@ const cryptopolicies = {
             {_("Learn more")}
         </Button>
     </Flex>),
+    "FIPS:OSPP": _("FIPS with further Common Criteria restrictions"),
```

Comment:
Missing dot at the end of the sentence. Also I think we want to explain this better than just "further Common Criteria restrictions".

Reply:
The dot is missing in all new added ones. Can you please add it everywhere?